### PR TITLE
Sets RUNTIME_DIR in runtimes, instead of openjdk base image.

### PR DESCRIPTION
**Pull request by joaoandremartins**
Helps fixing #64

Used in conjunction with https://github.com/GoogleCloudPlatform/jetty-runtime/pull/164

Will also require a change in format-env-appengine-vm.sh

### CODE REVIEW

File: openjdk8/src/main/docker/setup-env.d/20-debug-env.bash [hunk 1]
```diff
@@ -8,5 +8,4 @@ fi
 
 if [ "$DBG_ENABLE" = "true" ]; then
   unset CDBG_DISABLE
-  DBG_AGENT="$( RUNTIME_DIR=$JETTY_BASE /opt/cdbg/format-env-appengine-vm.sh )"
```
Comment (meltsufin) [inline openjdk8/src/main/docker/setup-env.d/20-debug-env.bash]: This is completely removing the debugger. I don't think we want to do that.

File: openjdk8/src/main/docker/setup-env.d/20-debug-env.bash [hunk 2]
```diff
@@ -8,5 +8,5 @@ fi
...
+  DBG_AGENT="$(/opt/cdbg/format-env-appengine-vm.sh)"
```
Comment (meltsufin) [inline openjdk8/src/main/docker/setup-env.d/20-debug-env.bash]: I guess it can now just be
`DBG_AGENT="/opt/cdbg/format-env-appengine-vm.sh"`

Comment (joaoandremartins) [inline openjdk8/src/main/docker/setup-env.d/20-debug-env.bash]: I think you still want `/opt/cdbg/format-env-appengine-vm.sh` to be evaled, no?

Comment (meltsufin) [inline openjdk8/src/main/docker/setup-env.d/20-debug-env.bash]: Right, never mind.

Comment (joaoandremartins) [inline openjdk8/src/main/docker/setup-env.d/20-debug-env.bash]: $ export DBG_AGENT="$(/usr/local/google/home/joaomartins/Downloads/cdbg_java_agent/format-env-appengine-vm.sh)"
$ echo $DBG_AGENT
-agentpath:/usr/local/google/home/joaomartins/Downloads/cdbg_java_agent/cdbg_java_agent.so=--log_dir=/var/log/app_engine,--logtostderr=false,--cdbg_extra_class_path=/webapps/root/WEB-INF/classes:/webapps/root/WEB-INF/lib

(without the format-env-appengine-vm.sh fix)

**Comment by joaoandremartins** [comment]
This change is pending internal change 155509870 from being submitted and published.

**Pull request merged**